07_Train_With_AML_Pipeline.ipynb

```diff
@@ -19,8 +19,10 @@
     "- [create the pipeline best model step](#aml_pipeline_bm_step),\n",
     "- [create the pipeline register model step](#aml_pipeline_rm_step),\n",
     "- [create the pipeline itself](#create_aml_pipeline),\n",
-    "- [publish the pipeline](#publish_aml_pipeline), and\n",
-    "- [run the published pipeline using its REST endpoint](#run_publish_aml_pipeline).\n",
+    "- [publish the pipeline](#publish_aml_pipeline),\n",
+    "- [run the published pipeline using its REST endpoint](#run_publish_aml_pipeline),\n",
+    "- [schedule the published pipeline to regularly run](#schedule_aml_pipeline), and\n",
+    "- [write out the pipeline's URI and key](#write_uri_key)\n",
```

Comment:
The published happy path deliberately does not create a scheduled pipeline. Doing so can leads to user problems and unexpected user costs. In addition, instrumentation should not be in the published path. These would be fine for the instrument branch.